views/js/runner/plugins/tools/itemThemeSwitcher/itemThemeSwitcher.js

```diff
@@ -74,6 +86,18 @@ define([
                 return themesConfig && _.size(themesConfig.available) > 1;
             }
 
+            function reloadThemes() {
+                var themeConfig = themeHandler.get('items');
```

Comment:
Please don't use the same variable name than in a upper scope. Didn't you run ESLint  on your code (this should have triggered a warning).